Performance/perf_runner.sh

```diff
@@ -106,10 +106,14 @@ function run_harness_and_concatenate_results() {
     "$language": {
 EOF
 
+  echo "Running $language test harness alone..."
+  sleep 3
+  DYLD_LIBRARY_PATH="$script_dir/_generated" "$harness" "$partial_results"
```

Comment:
+1, this is probably the right thing to do. It likely won't affect the comparative nature of the timings, but it'll give us truer absolute numbers.